pulumi/infra/grapl_consul_acls.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict, Optional
+
+import pulumi_consul as consul
+
+import pulumi
+
+
+class GraplConsulAcls(pulumi.ComponentResource):
+
+    ANONYMOUS_TOKEN_ID = "00000000-0000-0000-0000-000000000002"
```

Comment:
It would be good to add some documentation here with a link to https://www.consul.io/docs/security/acl/acl-system#builtin-tokens; I thought this was something you were making up yourself.

Documenting why we're using the anonymous token would also be useful.